### DISCUSSION TITLE
ASIO driver 闪退

### DISCUSSION THREAD

**Discussion by DAYANXX** [sist-cs120/project-wiki#15] [Project 1 Q&A] [answered]
When I run the example project, the ASIO driver appear in the task bar, and once the project is built, ASIO driver disappear.

**Comment by yhyan96** [comment, marked as answer]
Is there any error message?

You can try [RTL UTILITY](https://oblique-audio.com/rtl-utility.php), which is a more reliable tool, to determine which part causes the problem, the ASIO driver or the others (JUCE, example program, ...).

**Reply by DAYANXX** [reply]
In the AudioLatencyDemo, when I run the project, before AudioLatencyDemo application appear, I can see ASIO driver in the task bar. once AudioLatencyDemo application appear, ASIO driver disappear.

**Reply by yhyan96** [reply]
I have met this problem just a moment ago. It seems because the default device of the ASIO driver is a Bluetooth audio device, e.g., Bluetooth earphones, and this Bluetooth device is not connected. There are three ways I have tested to solve it.

1. Connect to the Bluetooth audio device.

If you have many Bluetooth audio devices and you don't know which one is the default device, it is troublesome to test them one by one. Then you can try the second way.

2. After running the project, click the ASIO driver icon on the taskbar (it may be hidden), as soon as possible, again and again, until the control panel appears. Then turn off the Bluetooth audio device and turn on your preferred device.

3. If you still failed, the last way is to delete all your Bluetooth audio devices.

**Reply by DAYANXX** [reply]
I solved it by 2, thank you very much.
